nomium/prometheus/src/lib.rs

```diff
@@ -0,0 +1,166 @@
+use dotenvy::dotenv;
```

Comment:
Ensure environment variables like TPROXY_METRICS_IP are set and documented for all deployment environments.